### PR TITLE
Healthcheck url extension

### PR DESCRIPTION
**Pull request by waveywaves**
Allow users to pass the healtcheck url extension where github actions can check the health of the preview

### CODE REVIEW

File: .github/workflows/reusable.yaml [hunk 1]
```diff
@@ -223,7 +228,7 @@ jobs:
         if: ${{ steps.find-deployment.outputs.id == '' && github.event.action != 'closed' && steps.url-auth-for-create.outputs.enabled == '' }}
         uses: docker://curlimages/curl:7.84.0
         with:
-          args: --retry 12 --retry-all-errors --retry-delay 0 --fail --location ${{ steps.create-preview.outputs.url }}
+          args: --retry 12 --retry-all-errors --retry-delay 0 --fail --location ${{ steps.create-preview.outputs.url }}${{ inputs.healthcheck-url-extension }}
```
Comment (axisofentropy) [inline .github/workflows/reusable.yaml:231]: Update line 236 too!

File: README.md [hunk 2]
```diff
@@ -127,6 +127,10 @@ Note that if this is an emtpy string, the reusable workflow will delete the prev
 
 (Optional) If you're controlling access to the URLs of your Preview Environments, set the credentials here so the workflow can confirm successful deployment.
 
+#### `healthcheck-url-extension`
+
+URL extension that will be appended to the preview URL where the reusable workflow will check for the health of the preview.
```
Comment (axisofentropy) [inline README.md:132]: Please add a note that this path should begin with a `/`.  Also add "Use this if the root URL of your application does not return an HTTP response code below 400."

**Comment by axisofentropy** [comment]
Also we'll pretty soon want to merge this into the `deprecate_set-output` branch or vice-versa.

File: README.md (hunk 2 above)
Comment (waveywaves) [reply README.md]: Updated

**Pull request merged**